lib/ts/index.ts

```diff
@@ -100,4 +137,9 @@ export let attemptRefreshingSession = AuthHttpRequest.attemptRefreshingSession;
 export let doesSessionExist = AuthHttpRequest.doesSessionExist;
 export let addAxiosInterceptors = AuthHttpRequest.addAxiosInterceptors;
 export let signOut = AuthHttpRequest.signOut;
+export const validateClaims = AuthHttpRequest.validateClaims;
 export { RecipeInterface, InputType };
+
+export { ClaimValidationError, ClaimValidationResult, SessionClaimValidator } from "./types";
+export { PrimitiveClaim } from "./claims/primitiveClaim";
+export { BooleanClaim } from "./claims/booleanClaim";
```

Comment:
should export them as "supertokens-website/claims"

Reply:
On second thought, this is not needed because when importing supertokens-website via a script tag, im not sure how users would use these claims. 

On this subject, even if these are here, how will users use this when importing via script tag?

Reply:
`supertokens.BooleanClaim`